src/redisearch_api.c

```diff
@@ -55,6 +55,24 @@ void RediSearch_DropIndex(IndexSpec* sp) {
   RWLOCK_RELEASE();
 }
 
+char **RediSearch_IndexGetStopwords(IndexSpec* sp) {
+  return GetStopWordsList(sp->stopwords);
```

Comment:
We do not keep the size? If we do I think its better to return it as an out param other then trust on NULL termination.

Reply:
We also use length in the setter `RediSearch_IndexOptionsSetStopwords` in issue #2002.
I suggest to also align the names of the getter and the setter to start with `RediSearch_IndexOptions`
e.g., as in `RediSearch_IndexOptionsSetStopwords`, `RediSearch_IndexOptionsSetGCPolicy`, etc.

Reply:
@oshadmi the function names you mention are for the option used to initialize the index and not the index itself

Reply:
We get the size from cardinality and we assert it matches the actual number of elements. :100: